### PR TITLE
install-mode docs

### PR DESCRIPTION
**Pull request by ibrokethecloud**
PR adds a more detailed doc for need for install mode and how to leverage pre-installed install mode raw images.

### CODE REVIEW

File: docs/install/install-binaries-mode.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* node password
+
+Once this information is made available, the installer will apply the endpoint info and bootstrap harvester. No further reboots will be required.
+
+### stream disk mode
+Harvester now publishes a raw image artifact for pre-installed harvester. An additional feature has been built into the harvester installer to allow streaming this pre-installed image directly to disk.
+
+This allows better integration with cloud providers.
+
+On `Equinix Metal` the streaming mode can be leverage with the following kernel arguments:
```
Comment (vickyhella) [inline docs/install/install-binaries-mode.md]:
```suggestion
On `Equinix Metal` the streaming mode can be leveraged with the following kernel arguments:
```

File: docs/install/install-binaries-mode.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+When the user boots this "configured" node, the console will wait for the end user to configure the following:
+
+* create/join option for harvester
+* management network interface details
+* cluster token
+* node password
+
+Once this information is made available, the installer will apply the endpoint info and bootstrap harvester. No further reboots will be required.
+
+### stream disk mode
```
Comment (bk201) [inline docs/install/install-binaries-mode.md:40]:
```suggestion
### Stream disk mode
```

File: docs/install/install-binaries-mode.md (hunk 1 above)
Comment (bk201) [inline docs/install/install-binaries-mode.md:45]:
```suggestion
On `Equinix Metal` the streaming mode can be leveraged with the following kernel arguments:
```

File: docs/install/install-binaries-mode.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+</head>
+
+_Available as of v1.2.0_
+
+`Install Harvester binaries only` mode allows end users to install and configure binaries and is better suited for cloud and edge use cases.
+
+![choose-installation-mode.png](/img/v1.2/install/choose-installation-mode.png)
+
+### Background
+Currently when a new harvester node is launched it needs to be the first node in the cluster or join an existing cluster.
```
Comment (bk201) [inline docs/install/install-binaries-mode.md:25]:
```suggestion
Currently when a new Harvester node is launched it needs to be the first node in the cluster or join an existing cluster.
```
Do we want to make the upper case first letter of Harvester for consistency?

File: docs/install/install-binaries-mode.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+* management network interface details
+* cluster token
+* node password
+
+Once this information is made available, the installer will apply the endpoint info and bootstrap harvester. No further reboots will be required.
+
+### Stream disk mode
+Harvester now publishes a raw image artifact for pre-installed harvester. An additional feature has been built into the harvester installer to allow streaming this pre-installed image directly to disk.
+
+This allows better integration with cloud providers.
```
Comment (vickyhella) [inline docs/install/install-binaries-mode.md:43]:
```suggestion
Harvester has published a raw image artifact for pre-installed Harvester. The Harvester installer now allows streaming a pre-installed image directly to disk to support better integration with cloud providers.
```

Comment (vickyhella) [reply docs/install/install-binaries-mode.md:43]: Better give a link to the `raw image artifact`, if any

File: docs/install/install-binaries-mode.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+On `Equinix Metal` the streaming mode can be leveraged with the following kernel arguments:
```
Comment (vickyhella) [inline docs/install/install-binaries-mode.md:45]:
```suggestion
On `Equinix Metal`, you can use the following kernel arguments to use the streaming mode:
```

File: docs/install/install-binaries-mode.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+```
+ip=dhcp net.ifnames=1 rd.cos.disable rd.noverifyssl root=live:http://${artifactEndpoint}/harvester-v1.2.0-rootfs-amd64.squashfs harvester.install.automatic=true harvester.scheme_version=1 harvester.install.device=/dev/vda  harvester.os.password=password harvester.install.raw_disk_image_path=http://${artifactEndpoint}/harvester-v1.2.0-amd64.raw harvester.install.mode=install console=tty1 harvester.install.tty=tty1 harvester.install.config_url=https://metadata.platformequinix.com/userdata harvester.install.management_interface.interfaces="name:enp1s0" harvester.install.management_interface.method=dhcp harvester.install.management_interface.bond_options.mode=balance-tlb harvester.install.management_interface.bond_options.miimon=100
+```
+
+:::note
+When leveraging streaming of disk, it is recommended to host the raw disk artifact closer to the targets. The raw disk artifact is nearly 16G in size.
```
Comment (vickyhella) [inline docs/install/install-binaries-mode.md:52]:
```suggestion
When streaming to disk, it is recommended to host the raw disk artifact closer to the targets, as the raw disk artifact is nearly 16G in size.
```

**Review by vickyhella — CHANGES_REQUESTED** [review]
Some small suggestions on the grammar.

**Review by LucasSaintarbor — COMMENTED** [review]
+1 Vicky's suggestions

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

File: docs/install/install-binaries-mode.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 8
+sidebar_label: Install Binaries Only
+title: "Install Binaries Only Mode"
```
Comment (vickyhella) [inline docs/install/install-binaries-mode.md:4]:
```suggestion
sidebar_label: Install Harvester Binaries Only
title: "Install Harvester Binaries Only"
```

Comment (vickyhella) [reply docs/install/install-binaries-mode.md:4]: Modify the title to stay consistent with the UI.

**Review by vickyhella — APPROVED** [review]
LGTM!

**Review by LucasSaintarbor — APPROVED** [review]
LGTM, thanks!

**Pull request merged**